financial-connections/src/main/java/com/stripe/android/financialconnections/features/manualentrysuccess/ManualEntrySuccessScreen.kt

```diff
@@ -3,353 +3,30 @@
-            .border(
-                border = BorderStroke(1.dp, FinancialConnectionsTheme.colors.borderDefault),
-                shape = shape
-            )
-    ) {
-        Column(
-            Modifier
-                .padding(start = 16.dp, end = 16.dp, top = 16.dp)
-        ) {
-            val titleColor = FinancialConnectionsTheme.colors.textSecondary
-            val tableData =
-                buildTableRows(microdepositVerificationMethod)
-            last4?.let {
-                Row(
-                    verticalAlignment = Alignment.CenterVertically,
-                    horizontalArrangement = Arrangement.spacedBy(8.dp)
-                ) {
-                    Icon(
-                        painter = painterResource(id = R.drawable.stripe_ic_bank),
-                        tint = FinancialConnectionsTheme.colors.textSecondary,
-                        contentDescription = "Bank icon"
-                    )
-                    Text(
-                        text = stringResource(R.string.stripe_manualentrysuccess_table_title, it),
-                        style = FinancialConnectionsTheme.typography.bodyCode.copy(color = titleColor)
-                    )
-                }
-                Spacer(Modifier.size(8.dp))
-            }
-            Row {
-                TitleCell(text = "Transaction")
-                TitleCell(text = "Amount")
-                TitleCell(text = "Type")
-            }
-            Divider(
-                modifier = Modifier.padding(top = 4.dp, bottom = 8.dp),
-                color = FinancialConnectionsTheme.colors.borderDefault
-            )
-            tableData.withIndex().forEach { (index, item) ->
-                val (transaction, amount, type) = item
-                val highlight = tableData.lastIndex != index
-                Row(Modifier.fillMaxWidth()) {
-                    TableCell(text = transaction.first, transaction.second, highlight)
-                    TableCell(text = amount.first, amount.second, highlight)
-                    TableCell(text = type.first, type.second, highlight)
-                }
-            }
-        }
-        // Bottom fade.
-        Box(
-            modifier = Modifier
-                .fillMaxWidth()
-                .height(26.dp)
-                .align(Alignment.BottomCenter)
-                .background(
-                    brush = Brush.verticalGradient(
-                        colors = listOf(
-                            FinancialConnectionsTheme.colors.textWhite.copy(alpha = 0f),
-                            FinancialConnectionsTheme.colors.textWhite.copy(alpha = 1f)
-                        )
-                    )
-                )
+    state.value.payload()?.let { payload ->
+        SuccessContent(
+            overrideAnimationForPreview = false,
+            completeSessionAsync = state.value.completeSession,
+            payload = payload,
+            onDoneClick = { viewModel.onSubmit() },
```

Comment:
Nit: Could be a method reference.

Reply:
https://github.com/stripe/stripe-android/pull/7761/commits/8f51697d869ccf1ddb4062328acb5c1bbc96b612